doc/classes/AudioEffectCompressor.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectCompressor" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Compressor audio effect to an Audio bus.
+		Reduces sounds that exceed a certain threshold level, smooths out the dynamics and increases the overall volume.
 	</brief_description>
 	<description>
+		Dynamic range compressor reduce the level of the sound when the amplitude goes over a certain threshold in Decibels. One of the main uses of a compressor is to increase the dynamic range while clipping the least possible (when sound goes over 0dB).
+		Compressor has may uses in the mix :
+		- Uses in the Master bus to compress the whole output (Although a Limiter is probably better)
```

Comment:
"uses" is implicit in the list, so you don't have to repeat it:
"- In the Master bus..."

Link to Limiter so users can click to see why it may be better